bottomless/src/replicator.rs

```diff
@@ -276,13 +241,10 @@ impl Replicator {
         }
 
         let db_path = db_path.into();
-        let db_name = {
-            let db_id = options.db_id.unwrap_or_default();
-            let name = match db_path.find('/') {
-                Some(index) => &db_path[..index],
-                None => &db_path,
-            };
-            db_id + ":" + name
```

Comment:
I removed this part - it's a legacy thing that binds part of the database path into a S3 bucket keyspace. It's not necessary as an unique db_id is always provided by sqld, and it would make arbitrary restoration harder.